messages/crypto.json

```diff
@@ -0,0 +1,3 @@
+{
+  "MacKeychainOutOfSync": "\nWe’ve encountered an error with the Mac keychain being out of sync with your `sfdx` credentials. To fix the problem, sync your credentials by logging into your Dev Hub"
```

Comment:
I'm assuming this has been reviewed already, but it might be worth it be very specific on how. I wouldn't put it past a customer to ask. `...logging into your Dev Hub using the auth commands.`

Reply:
I think it was a deliberate choice from doc to avoid telling them what commands to use

Reply:
I figured but figured "auth commands" was still generic enough. i.e. logging into your Dev Hub via the browser (without running a command) won't do anything, which a customer can read it as that.